the-blue-alliance-ios/View Elements/Awards/AwardTeamButton.swift

```diff
@@ -0,0 +1,117 @@
+import Foundation
+import UIKit
+
+class AwardTeamButton: UIView {
+
+    var widthRange: ClosedRange<CGFloat> {
+        didSet {
+            invalidateIntrinsicContentSize()
+        }
+    }
+    var headerText: String? {
+        get { return headerLabel.text }
+        set {
+            headerLabel.text = newValue
+            invalidateIntrinsicContentSize()
+        }
+    }
+    var subheaderText: String? {
+        get { return subheaderLabel.text }
+        set {
+            subheaderLabel.text = newValue
+            invalidateIntrinsicContentSize()
+        }
+    }
+    private let headerLabel = UILabel()
+    private let subheaderLabel = UILabel()
+    private let arrowView = UIImageView(image: UIImage(named: "ic_arrow")?.withRenderingMode(.alwaysTemplate))
```

Comment:
I think we should be able to set these arrows as template images in the asset catalog - let me know if you need help, but it'll be in the right-hand pane. I forget what the check box is.